core/src/main/java/com/novoda/noplayer/internal/exoplayer/NoPlayerExoPlayerCreator.java

```diff
@@ -87,10 +86,6 @@ ExoPlayerTwoImpl create(Context context,
                     allowCrossProtocolRedirects
             );
 
-            MediaCodecSelector mediaCodecSelector = downgradeSecureDecoder
```

Comment:
We pass the boolean down to where it is used now. It goes to a factory that handles the construction and usage of this.